src/psplogger.py

```diff
@@ -28,20 +28,22 @@ def __init__ (self, msg : str, level: str = "debug", tweetID : str = "", tweet :
         else:
             self.tweetID = str(tweetID)
 
-class ErrorEntry:
-    level : str = ""
-    location : str = ""
-    module : str = ""
-    errormsg : str = ""
-    time : str = ""
-    tweetID : str = ""
-    def __init__ (self, errormsg : str, level: str = "error", location : str = "", module : str = "", tweetID : str = ""):
-        self.level = level
-        self.location = location
-        self.module = module
-        self.errormsg = errormsg
-        self.time = datetime.datetime.utcnow().isoformat() + 'Z'
-        self.tweetID = str(tweetID)
+class ErrorEntry(LogEntry):
+    # Campos específicos del Error (Exception)
+    #location : str = ""
+    #module : str = ""
+    #errormsg : str = ""
+
+    def __init__ (self, errormsg : str, location : str = "", module : str = "", tweetID : str = ""):
+        #self.location = location
+        #self.module = module
+        #self.errormsg = errormsg
+        #self.tweetID = str(tweetID)
```

Comment:
Don't comment them out, just delete them, we can still find them in the repo history.